client/reader/following-manage/index.jsx

```diff
@@ -140,6 +142,25 @@ class FollowingManage extends Component {
 		);
 	};
 
+	reportFollowByUrlRender = () => {
+		const siteUrl = this.props.readerAliasedFollowFeedUrl;
+		const showingFollowByUrlButton = this.shouldShowFollowByUrl();
+
+		if ( siteUrl && showingFollowByUrlButton ) {
+			recordTrack( 'calypso_reader_following_manage_follow_by_url_render', {
+				url: siteUrl,
+			} );
+		}
+	};
+
+	shouldShowFollowByUrl = () => resemblesUrl( this.props.sitesQuery );
```

Comment:
Not that I know of!

 I had a PR out with a "ready to merge" but it was considered an alternative to displaying both search results + the follow by url button.

https://github.com/Automattic/wp-calypso/pull/13957